yt_dlp/extractor/rule34video.py

```diff
@@ -50,12 +52,14 @@ def _real_extract(self, url):
         title = self._html_extract_title(webpage)
         thumbnail = self._html_search_regex(r'preview_url:\s+\'([^\']+)\'', webpage, 'thumbnail', default=None)
         duration = self._html_search_regex(r'"icon-clock"></i>\s+<span>((?:\d+:?)+)', webpage, 'duration', default=None)
+        tags = re.findall(r'<a class="tag_item"\s+href="https://rule34video\.com/tags/\d+/".*>(?P<tag>.*)</a>', webpage)
```

Comment:
idk if tags can have special chars, but wrap this in `unescapeHTML` just in case.

Reply:
Done